app/scripts/app.js

```diff
@@ -17,7 +23,8 @@ blocChat.config(['$stateProvider', '$locationProvider', function($stateProvider,
 
 blocChat.factory("chatRooms", [
     "$firebaseArray",
-    function($firebaseArray) {
+    "CurrentRoom",
+    function($firebaseArray, CurrentRoom) {
         var roomsUrl = 'https://stewart-bloc-chat.firebaseio.com/rooms';
         var roomsRef = new Firebase(roomsUrl);
         var roomsObject = $firebaseArray(roomsRef);
```

Comment:
I actually ended up solving this issue—looks like my earlier notes got collapsed somehow.

Anyway, I was just wondering if you had general comments about best practices that I could apply to the code I wrote for this commit.